matchzoo/models/cdssm_model.py

```diff
@@ -83,18 +81,15 @@ def build(self):
 
         CDSSM use Siamese arthitecture.
         """
-        input_shape_left = self._params['input_shapes'][0]
-        input_shape_right = self._params['input_shapes'][1]
-        base_network_left = self._create_base_network(
-            input_shape=input_shape_left)
-        base_network_right = self._create_base_network(
-            input_shape=input_shape_right)
+        input_shape = self._params['input_shapes'][0]
+        base_network = self._create_base_network(
+            input_shape=input_shape)
         # Left input and right input.
-        input_left = Input(name='id_left', shape=input_shape_left)
-        input_right = Input(name='id_right', shape=input_shape_right)
+        input_left = Input(name='id_left', shape=input_shape)
+        input_right = Input(name='id_right', shape=input_shape)
```

Comment:
Are `input_left` and `input_right` both use `the input_shape[0]`?  Usually, each one has its own size.

Reply:
`input_left.shape = (None, 90000)`, `input_right.shape=(None, 90000)`
`None` denotes for variable input shape, better than using a `FixedLenghtProcessUnit`.